deploy/operator/libvirt_disks.sh

```diff
@@ -41,12 +44,11 @@ function destroy() {
         done
     done
 
+    oc get pv -o=name | xargs -r kubectl delete
```

Comment:
but it's not the same abstraction level
not sure it should be in ``libvirt_disks.sh`` script